### PR TITLE
cloud-armor: Add dashboard for Cloud Armor (formerly Network Security)

### PR DESCRIPTION
**Pull request by SamiSousa**
Note: This GCP product was formerly known as Network Security.

### CODE REVIEW

File: dashboards/cloud-armor/cloud-armor-policies-overview-01.png
Comment (EvanSimpson) [inline dashboards/cloud-armor/cloud-armor-policies-overview-01.png:1]: Can you retake these screenshots with a title that matches the display name in the JSON file?

File: dashboards/cloud-armor/cloud-armor-policies-overview.json [hunk 1]
```diff
@@ -0,0 +1,609 @@
...
+  },
+  "dashboardFilters": [
+    {
+      "labelKey": "policy_name",
+      "templateVariable": "",
+      "filterType": "RESOURCE_LABEL",
+      "valueType": "STRING"
+    }
+  ],
+  "labels": {}
```
Comment (EvanSimpson) [inline dashboards/cloud-armor/cloud-armor-policies-overview.json:608]: nit: remove

File: dashboards/cloud-armor/cloud-armor-policies-overview.json [hunk 2]
```diff
@@ -0,0 +1,609 @@
...
+    ]
+  },
+  "dashboardFilters": [
+    {
+      "labelKey": "policy_name",
+      "templateVariable": "",
+      "filterType": "RESOURCE_LABEL",
+      "valueType": "STRING"
+    }
+  ],
```
Comment (EvanSimpson) [inline dashboards/cloud-armor/cloud-armor-policies-overview.json:607]: nit: move this up top, between displayName and mosaicLayout

File: dashboards/cloud-armor/cloud-armor-policies-overview.json [hunk 3]
```diff
@@ -0,0 +1,609 @@
...
+                  "unitOverride": "",
+                  "outputFullDuration": false
+                },
+                "plotType": "LINE",
+                "legendTemplate": "Total Requests",
+                "minAlignmentPeriod": "60s",
+                "targetAxis": "Y1",
+                "dimensions": [],
+                "measures": [],
+                "breakdowns": []
```
Comment (EvanSimpson) [inline dashboards/cloud-armor/cloud-armor-policies-overview.json:34]: Can you go through and remove all fields that have empty array and empty string values?

File: dashboards/cloud-armor/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+sample_dashboards:
+  -
+    category: Cloud Armor
+    id: cloud-armor-policies-overview
+    display_name: Cloud Armor Policies Overview
+    description: |-
+      This dashboard is based on Google Cloud's Cloud Armor Policies (also known as Network Security Policies) and has charts displaying Requests for each policy, as well as a table of open incidents related to your Cloud Armor Policies.
+      There are also links to Cloud Armor Policies in Google Cloud Platform and docs for Observing your Cloud Armor Policies.
+    related_integrations:
+      - id: cloud-armor
```
Comment (EvanSimpson) [inline dashboards/cloud-armor/metadata.yaml:10]: Does this exist? Let's remove this or hold on submitting until it exists.

**Review by EvanSimpson — CHANGES_REQUESTED** [review]

File: dashboards/cloud-armor/metadata.yaml (hunk 4 above)
Comment (SamiSousa) [reply dashboards/cloud-armor/metadata.yaml]: It is being added, let's hold until it's submitted.

File: dashboards/cloud-armor/cloud-armor-policies-overview.json (hunk 3 above)
Comment (SamiSousa) [reply dashboards/cloud-armor/cloud-armor-policies-overview.json]: Done!

File: dashboards/cloud-armor/cloud-armor-policies-overview.json (hunk 2 above)
Comment (SamiSousa) [reply dashboards/cloud-armor/cloud-armor-policies-overview.json]: Moved!

File: dashboards/cloud-armor/cloud-armor-policies-overview.json (hunk 1 above)
Comment (SamiSousa) [reply dashboards/cloud-armor/cloud-armor-policies-overview.json]: Removed this. Should I also remove other instances of empty objects?

File: dashboards/cloud-armor/cloud-armor-policies-overview-01.png
Comment (SamiSousa) [reply dashboards/cloud-armor/cloud-armor-policies-overview-01.png:1]: Good call! Updated the screenshots

File: dashboards/cloud-armor/metadata.yaml (hunk 4 above)
Comment (SamiSousa) [reply dashboards/cloud-armor/metadata.yaml]: Cloud Armor integration has been added

**Review by EvanSimpson — APPROVED** [review]

**Pull request merged**